sinric/_sinricprosocket.py

```diff
@@ -61,5 +62,6 @@ async def receiveMessage(self, connection):
 
     async def handle(self, udp_client):
         while True:
+            sleep(1)                    
```

Comment:
Better off using:
```python
from asyncio import sleep
...
await sleep(0)
```
I think it's slightly better to move the sleep to after the `while queue.qsize...` loop too

Reply:
Having played with this for a bit having a non-zero sleep time also improve CPU usage significantly... I'm currently running with 0.1. Maybe the sleep should be a function of the initial qsize